.circleci/config.yml

```diff
@@ -598,7 +598,7 @@ jobs:
         default: large
       parallelism:
         type: integer
-        default: 5
+        default: 7
```

Comment:
Do you think this is needed, or do you think this is still needed, or do you think split by timings was most helpful for balancing out tests?

Reply:
I'll remove and give it a go 😬